dataset.py

```diff
@@ -64,13 +64,13 @@ class Dataset(object):
     When iterated over, this object returns (minibatch, minibatch target) pairs.
     """
     def __init__(self, directory, split=(0.6, 0.2, 0.2), minibatch_size=10,
-            pool_size=8, cache=False):
+            pool_size=8, cache=False, index_file):
         if not directory.endswith('/'):
             self.directory = '%s/' % directory
         else:
             self.directory = directory
-        path = '%spublic_list_primary_v3_full_21june_2017.csv' % self.directory
-        lines = open(path, 'r').readlines()[1:]
+
+        lines = open(index_file, 'r').readlines()[1:]
```

Comment:
This should be `lines = open('%s%s' % (self.directory, index_file), 'r').readlines()[1:]`

Reply:
I disagree with this. The zip file of data doesn't contain the .csv index file. They are downloaded separately. I want the user to provide the full path to the .csv file, not just the name of the file that is within the directory.

Reply:
Okay, fair enough!